README.md

```diff
@@ -55,7 +55,7 @@ performance will not be good. It might be a good idea to move this data to e.g.
    `git clone https://github.com/UCLALibrary/californica.git; cd californica`
 1. Install dependencies
    `bundle install`
-1. Setup your database.
+1. Setup your database. SEE THE SECTION IMMEDIATELY FOLLOWING FOR MORE DETAILS!
```

Comment:
Could we link to `#important-details-for-postgresql-security-and-configuration` instead of including this text?

I wonder whether we should move the postgres configuration info somewhere other than the README.md.

Reply:
I'm open to all kinds of alternate wording ideas. The gist of it is: I *like* the code changes here, I'm totally +1 on them, they help me get tests in my environment working. The documentation changes are secondary to that. Maybe we move all the docs for database setup to the wiki, and point from the README to the wiki? That might help us get past the upcoming DB change, too.